docs/process/Estimation-Process.md

```diff
@@ -27,73 +27,130 @@ below the estimate. These assumptions translate to risks - because if
 they turn out to be incorrect, then they will add more time to the
 task.
 
-It's also possible to provide confidence intervals around an
-estimation to quantify the uncertainty. For example, a development
-task could be estimated to take 30 hours expected, +20 hours worst
-case, -5 hours best case. However even then, how often do we expect
-that actual time will come in longer than "worst case?"
+## Capturing uncertainty in our estimates
 
-## Story points estimates
+One common way to express an estimate for a given task is to give a single
+number: the number of remaining days of effort the task owner believes that he
+or she will require to finish that task.
 
-We accept that accurate estimation is hard, for multiple reasons, so
-instead adopt the "just in time" planning technique from XP.
+However, a single number alone is unable to capture the degree of confidence or
+uncertainty the author of an estimate has. To allow estimate authors to record
+their confidence or uncertainty, we need a bit more information.
 
-This uses unitless "story points" for estimation of each
-ticket. Possible story point values are on the Fibonacci sequence:
-1, 2, 3, 5, 8, 13.
+Therefore, we attach to each task a **pair** of estimates:
 
-The idea is that these point values represent task size in an abstract
-way. An estimate in time for a task can be calculated from the
-estimate in story points divided by the project velocity, adjusted
-according to staff availability, other factors, etc.
+- **Most-probable estimate (days)**: the number of remaining days of effort the
+  task owner believes he or she will most likely need to finish that task.
 
-The decision on whether a task is included in the next sprint is
-based on its points estimate and the project velocity.
+- **Most-pessimistic estimate (days)**: the number of remaining days of effort
+  the task owner believes he or she will need to be 90% of confident of
```

Comment:
I would prefer to stay _well away_ from assigning numbers to confidence intervals. Terms like "quite" or "very" or "roughly" are far more appropriate I think.
Same goes with any pseudo-mathematical language, or using these numbers to calculate other estimates.
We simply do not have the historical data and estimation skill for that (... yet?).

Reply:
I initially had the same feeling that the "90%" number in the confidence interval would merely give an illusion of precision, but after chatting with Jonathan, I have also come to believe that a more qualitative formulation like "very confident" does not capture the intent either.

The idea is this: Even though we may not (yet or ever) have the historical data or skill to estimate that number, stating that we want to estimate the "90%" confidence interval is something that we *could*, at least in principle, compare to historical data. Put differently: Even though my gut feeling for the pessimistic estimate will always be the number of days that I feel "very confident" about, the more precise "90% confident" qualifier encourages me to anchor the specific number of days to a timeframe were indeed, 90% of the time, I will have been historically able to finish the task in fewer days. The purpose of the confidence statement is not to reflect my own confidence about the number that I estimated here, but to reflect the timeframe for which my estimate is supposed to be — even though my estimate for that may just be a wild guess for the time being.

I will try to change the text to make this more clear. I do feel that the phrase "I'm 90% confident" is semantically not well-defined (feeling about my confidence VS unit of measurement).